### PR TITLE
AV-1496 As a product owner I want the design of api collection implemented

### PR DESCRIPTION
**Pull request by bertofl90**

### CODE REVIEW

File: modules/ckanext-ytp_main/ckanext/ytp/templates/package/snippets/resources_list3.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      {% block resource_list_inner %}
+      {% for resource in resources %}
+        {% snippet 'package/snippets/resource_item3.html', pkg=pkg, res=resource, draft=draft or false, resource_under_edit=resource_under_edit or false %}
+      {% endfor %}
+      {% endblock %}
+    </ul>
+  {% else %}
+    <p class="empty">
+      {% if h.check_access('package_update', {'id':pkg.id }) %}
+        {# Comment out "add some" as action doesn't exist yet #}
```
Comment (beazil) [inline modules/ckanext-ytp_main/ckanext/ytp/templates/package/snippets/resources_list3.html:16]: Is this comment here for a purpose or should it be removed?

File: modules/ytp-assets-common/src/less/ckan/upstream-overrides.less [hunk 2]
```diff
@@ -12,6 +12,18 @@ body {
   }
 }
 
+#content {
+  background: #fafafa;
+}
+
+#content .wrapper {
+  background: #fff;
```
Comment (beazil) [inline modules/ytp-assets-common/src/less/ckan/upstream-overrides.less:20]: There is a variable for this `@suomi-whiteBase`

File: modules/ytp-assets-common/src/less/components/list--resource.less [hunk 3]
```diff
@@ -24,18 +28,45 @@
...
       font-weight: 600;
       margin-left: auto;
     }
 
+    .resource-item__info__title {
+      color: @opendata-brand-primary;
+      transition: color @transition-duration--short @transition-timing-function--default;
+      font-size: 16px;
+      font-weight: normal;
+      line-height: 24px;
```
Comment (beazil) [inline modules/ytp-assets-common/src/less/components/list--resource.less:41]: Also here we might have used variables, at least for font-weight

File: modules/ytp-assets-common/src/less/components/list--resource.less [hunk 4]
```diff
@@ -24,18 +28,45 @@
...
+      font-family: @font-family-sans-serif;
+    }
+
     a {
       color: @opendata-brand-primary;
       transition: color @transition-duration--short @transition-timing-function--default;
+      font-size: 16px;
+      font-weight: 600;
+      line-height: 24px;
```
Comment (beazil) [inline modules/ytp-assets-common/src/less/components/list--resource.less:50]: Also here we might have used variables, at least for font-weight

**Review by beazil — CHANGES_REQUESTED** [review]
Looks good for me, some changes on less-files to make them use more variables (at least for colors and font-weights)

File: modules/ytp-assets-common/src/less/ckan/upstream-overrides.less [hunk 5]
```diff
@@ -12,6 +12,18 @@ body {
   }
 }
 
+#content {
+  background: #fafafa;
```
[the remark this answers is not in the document]
Comment (bertofl90) [reply modules/ytp-assets-common/src/less/ckan/upstream-overrides.less:16]: I've removed it

File: modules/ckanext-ytp_main/ckanext/ytp/templates/package/snippets/resources_list3.html (hunk 1 above)
Comment (bertofl90) [reply modules/ckanext-ytp_main/ckanext/ytp/templates/package/snippets/resources_list3.html]: it has been removed

**Pull request merged**